python/grpc/demo_grpc.py

```diff
@@ -0,0 +1,13 @@
+from ddtrace import Pin, patch
+import grpc
+
+# If not patched yet, you can patch grpc specifically
+patch(grpc=True)
+
+# This will report a span with the default settings
+with grpc.insecure_channel('localhost:50051') as channel:
+    stub = HelloStub(channel)
```

Comment:
Assuming we are missing the protobufs for `HelloStub`, etc?

Reply:
I can put them here also but I didn't see the point as it's the basic example from grpc website. Would you prefer a comment about that or a dedicated folder here ?

Reply:
It would be nice to have here so someone can just come to this directory, run this app as-is and have it work as expected.

Which means it would also be nice to have a `README.md` and `requirements.txt` as well

Reply:
Ok, to summarize, I'll have a .proto (and generated files), a server and a client. Also, a README and requirements will be added. Right ?